.github/workflows/e2e-ci.yml

```diff
@@ -128,6 +131,9 @@ jobs:
       - name: Make jest available
         working-directory: ${{env.working-directory}}
         run: cp ../../node_modules/.bin/jest node_modules/.bin/
+      - name: Check E2E wallet balance
+        working-directory: ${{env.working-directory}}/e2e/scripts
+        run: node --unhandled-rejections=strict ./check-e2e-wallet-balance.ts
```

Comment:
ditto about `ts-node`

Reply:
Fixed in https://github.com/valora-inc/wallet/commit/8ef142ea9af39f48f34bef82c345e71fb12d9de6!